src/version.js

```diff
@@ -0,0 +1,11 @@
+import validate from './validate.js';
+
+function version(uuid) {
+  if (validate(uuid)) {
+    return parseInt(uuid.substr(14, 1), 16);
+  }
+
+  return -1;
```

Comment:
Here I'm still a bit unsure whether -1 is the best return value for an invalid UUID… @broofa wdyt?

Reply:
@ctavan I would recommend always returning the same type from a function…